apps/web/src/pages/WorkforceAdjustment/WorkforceAdjustmentPage.tsx

```diff
@@ -0,0 +1,620 @@
+          </Instructions.Step>
+          <Instructions.Step img={{ src: stepThree, darkSrc: stepThreeDark }}>
+            <p className="mb-6 font-bold">
+              {intl.formatMessage({
+                defaultMessage: "3. Verify your work email address.",
+                id: "nyTeqb",
+                description:
+                  "Text for workfoce adjustment ->  verifying email step.",
+              })}
+            </p>
+            <p>
+              {intl.formatMessage({
+                defaultMessage:
+                  "This will unlock employee tools that will be accessible for as long as you’re a GC employee.",
+                id: "NfTF2V",
+                description:
+                  "Description of what verifying email does for users",
+              })}
+            </p>
+          </Instructions.Step>
+          <Instructions.Step img={{ src: stepFour, darkSrc: stepFourDark }}>
+            <p className="mb-6 font-bold">
+              {intl.formatMessage({
+                defaultMessage:
+                  "4. Complete the workforce adjustment section under Career Planning.",
+                id: "STEQs9",
+                description:
+                  "Text for workfoce adjustment -> completing workforce adjustment step.",
+              })}
+            </p>
+            <p>
+              {intl.formatMessage({
+                defaultMessage:
+                  "The workforce adjustment tools are found in your employee profile. Be sure to confirm all the details needed in this section.",
+                id: "iTOQmE",
+                description:
+                  "Instructions on where to find workforce adjustment tools",
+              })}
+            </p>
+          </Instructions.Step>
+          <Instructions.Step img={{ src: stepFive, darkSrc: stepFiveDark }}>
+            <p className="mb-6 font-bold">
+              {intl.formatMessage({
+                defaultMessage:
+                  "5. Add the relevant functional community to your profile.",
+                id: "kKs6C1",
+                description:
+                  "Text for workfoce adjustment -> adding communities step.",
+              })}
+            </p>
+            <p>
+              {intl.formatMessage({
+                defaultMessage:
+                  "This is essential if you want our team to be able to help you find a job or alternations match.",
+                id: "OltIDs",
+                description:
+                  "Description of why adding communities is important",
+              })}
+            </p>
+          </Instructions.Step>
+          <Instructions.Step img={{ src: stepSix, darkSrc: stepSixDark }}>
+            <p className="mb-6 font-bold">
+              {intl.formatMessage({
+                defaultMessage: "6. Wait for our team to reach out.",
+                id: "3LMOt/",
+                description: "Text for workfoce adjustment -> waiting step.",
+              })}
+            </p>
```

Comment:
Don't need the arrow after the final step I believe

<img width="630" height="414" alt="image" src="https://github.com/user-attachments/assets/5b488a1b-0650-4d73-8f6e-19e565243cf6" />

Reply:
Removed in [remove arrow for last step](https://github.com/GCTC-NTGC/gc-digital-talent/pull/14772/commits/2247a1a835ccd19f3209efcbc828accceb740006)